src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,61 @@
+package core.basesyntax;
+
+import core.basesyntax.bd.FruitStorage;
+import core.basesyntax.bd.dao.FruitDaoImpl;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.CreateReportService;
+import core.basesyntax.service.FruitShopTransferService;
+import core.basesyntax.service.ParseReaderService;
+import core.basesyntax.service.ReaderService;
+import core.basesyntax.service.WriterService;
+import core.basesyntax.service.impl.CreateReportServiceImpl;
+import core.basesyntax.service.impl.FruitShopTransferServiceImpl;
+import core.basesyntax.service.impl.ParseReaderServiceImpl;
+import core.basesyntax.service.impl.ReadFromFileCsvImpl;
+import core.basesyntax.service.impl.WriteToFileCsvImpl;
+import core.basesyntax.service.operationhandler.Operation;
+import core.basesyntax.service.operationhandler.OperationHandler;
+import core.basesyntax.service.operationhandler.impl.AddingOperationHandler;
+import core.basesyntax.service.operationhandler.impl.RemovingOperationHandler;
+import core.basesyntax.service.validation.Validator;
+import core.basesyntax.service.validation.ValidatorImpl;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.OperationStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String INPUT_DATA_FILE_PATH = "input.csv";
+    private static final String REPORT_FILE_PATH = "output.csv";
```

Comment:
Place the input and output files into the src/main/resources folder.

please see checklist